test/SeedCppCoreUnitTest/Tests/DAL/Translators/Db/DatabaseTranslatorTest.h

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include "SeedCppCore/DAL/Translators/Db/IDatabaseEntityTranslator.h"
+
+#include "TestUtilitiesInterface/EntityComparator.h"
+
+#include "DbAdapterTestUtilities/Stubs/StubTableRecord.h"
+#include "DbAdapterTestUtilities/Stubs/StubField.h"
+#include "DbAdapterTestUtilities/Stubs/StubFieldValue.h"
+
+
+using namespace testing;
+using namespace systelab::test_utility;
+using namespace systelab::db;
+using namespace systelab::db::test_utility;
+
+namespace seed_cpp { namespace unit_test {
+	
+	template <typename _DatabaseTranslatorTestWrapper>
+	class DatabaseTranslatorTest : public Test
+	{
+	public:
+		void SetUp()
+		{
+			m_translator = _DatabaseTranslatorTestWrapper::buildDbTranslator(m_model);
+		}
+
+		std::unique_ptr<StubTableRecord> buildRecord(const std::vector<StubFieldValue>& data)
+		{
+			std::vector<std::unique_ptr<StubFieldValue> > fieldValues;
+			for (auto cit = data.begin(); cit != data.end(); cit++)
```

Comment:
++cit ;)

Reply:
I think that compilers should be able to optimize this, but changes anyway.